interface_tester/collector.py

```diff
@@ -175,7 +175,7 @@ def _gather_charms_for_version(version_dir: Path) -> Optional[_InterfacesDotYaml
 
     providers = charms.get("providers") or []
     requirers = charms.get("requirers") or []
-    owners = charms.get("owners") or []
+    maintainers = charms.get("maintainers", "")
```

Comment:
```suggestion
    maintainers = charms.get("maintainers") or ""
```

so that on the off chance that someone puts a falsy value that is not "" (say, a `null`), we turn it to "" instead